Mathlib/Init/Data/Nat/Bitwise.lean

```diff
@@ -221,6 +227,11 @@ theorem shiftr_zero : ∀ n, shiftr 0 n = 0 := by
     rw [shiftr, div2, IH]
     rfl
 
+theorem shiftRight_eq_shiftr: n >>> m = Nat.shiftr n m := by
+  simp only [HShiftRight.hShiftRight, ShiftRight.shiftRight]
+  induction' m with m hm generalizing n
+  <;> simp [Nat.shiftr, Nat.shiftRight, Nat.div2_val, *]
```

Comment:
This looks to me like an indication that we should just delete `shiftr`, we don't need two ways to spell the same function. Can you make a separate PR to do that (and the same for `shiftLeft`?

Reply:
Sure!

Reply:
Ok [https://github.com/leanprover-community/mathlib4/pull/6356](here) it is.

Reply:
This can now be deleted.